packages/flutter_tools/lib/src/fuchsia/fuchsia_sdk.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../base/common.dart';
+import '../base/context.dart';
+import '../base/process.dart';
+
+/// The fuchsia SDK.
+FuchsiaSdk get fuchsiaSdk => context[FuchsiaSdk];
+
+/// The fuchsia SDK shell commands.
+///
+/// This workflow assumes development within the fuchsia source tree, providing
```

Comment:
The wording is a little confusing here (this class doesn't provide the fx command-line tool).  Maybe:

> This workflow assumes development within the Fuchsia source tree, including a working `fx` command-line tool in the user's `PATH`.